simple_network_sim/sampleUseOfModel.py

```diff
@@ -77,9 +77,8 @@ def runSimulation(
     if trials <= 1:
         # The averaging logic is slow and wastes a bunch of memory, skip it if we don't need it
         logger.info("Running simulation (1/1)")
-        disposableNetwork = copy.deepcopy(network)
```

Comment:
I suggest you don't delete this just yet. Maybe we can make the CLI changes in another PR.

Reply:
Done